src/main/java/com/beimin/eveapi/response/corporation/ContactListResponse.java

```diff
@@ -1,16 +1,29 @@
 package com.beimin.eveapi.response.corporation;
 
 import com.beimin.eveapi.model.shared.Contact;
-import com.beimin.eveapi.model.shared.NamedList;
+import com.beimin.eveapi.model.shared.ContactLabel;
 import com.beimin.eveapi.response.shared.AbstractContactListResponse;
+import java.util.Set;
```

Comment:
Have a look if your IDE has an switch to sort the imports on save.
java should be at the first place.